### ISSUE TITLE
Binder Link should point to head of master branch (tour can be a different binder link).

### ISSUE THREAD

**Issue by gutow** [jupyterlab/retrolab#264]
### Problem

I believe it is most useful for people to have the primary binder link point to the head of the master branch. This way people can test features that are in the master branch but not yet in releases without having to set it up a GIT repository on their own machine. This is especially useful for non- or casual coders, trying to evaluate if the offering is close to meeting their needs. I ran into this with checking if a bug fix that has been merged worked. As this is not one of my primary coding projects, I did not want to add its repository to my already massive list just to check.

More sophisticated users can also launch their own binder based on the master branch, but the key usefulness of binder is that people who are not familiar with the whole process can try things.

### Proposed Solution

* Have a primary binder link that loads the current master branch.
* Have a secondary binder link that loads a tour.
or possibly reverse the order.

**Comment by jtpio** [comment]
Thanks @gutow.

There is already a Binder link that points to the latest `main`:

![image](https://user-images.githubusercontent.com/591645/139573715-736c8c9d-84c2-498a-b364-a914a07acc53.png)

Maybe this could be made more explicit below in the the "Try it" section. For example we could rename that to "Try it online in your browser (stable)":

![image](https://user-images.githubusercontent.com/591645/139573790-351b4e14-fe93-46b3-b64d-0d98620ac0e3.png)

And have another one called "latest" or "unreleased".

Usually it's useful to have Binder links point to the latest stable version, to at least be able to use it as a reference.

**Comment by gutow** [comment]
My bad, I saw the usual "launch binder" and went with that. I also interpreted the red color of "launch main" to mean there was a problem as in "build passing" vs. "build failed". I should have figured it out. However, more casual users will not. I agree the solution is to have text explaining what each link does. Having both in the "Try it..." section would be good.

Something I also like to do is have a table of contents made of links to the anchors at the top to quickly navigate the readme. For example:
```
[Introduction](#introduction) | [Try It](#try-it-online-in-your-browser) | [Installation](#install) | ...
```
I've found this especially useful for people unfamiliar with GIT repositories, who do not realize they may have to scroll a lot to find all the information.
